packages/atlas/src/views/global/YppLandingView/YppLandingView.tsx

```diff
@@ -13,41 +14,46 @@ import { Text } from '@/components/Text'
 import { Button } from '@/components/_buttons/Button'
 import { SvgActionChevronR } from '@/components/_icons'
 import { useMediaMatch } from '@/hooks/useMediaMatch'
-import useScrollPosition from '@/hooks/useScrollPosition'
 
 import { BackImage, FrontImage, HeroImageWrapper, StyledLimitedWidthContainer } from './YppLandingView.styles'
 
 export const YppLandingView: FC = () => {
   const mdMatch = useMediaMatch('md')
   const smMatch = useMediaMatch('sm')
-  const frontImagePosition = useScrollPosition(smMatch ? 0.2 : 0.1)
+  const { ref: frontImageRef } = useParallax<HTMLImageElement>({
+    startScroll: 0,
+    endScroll: smMatch ? window.innerHeight / 3 : window.innerHeight,
+    translateY: [0, -15],
+  })
 
   return (
-    <StyledLimitedWidthContainer>
-      <LayoutGrid>
-        <GridItem as="header" colSpan={{ base: 12, sm: 8, lg: 6 }} colStart={{ sm: 3, lg: 4 }}>
-          <Text as="h1" variant={mdMatch ? 'h800' : 'h600'}>
-            Connect your YouTube channel & get paid
-          </Text>
-          <Text as="p" variant="t300" color="colorText" margin={{ top: 4, bottom: 8 }}>
-            Reupload and backup your YouTube videos to receive a guaranteed payout in the YouTube Partner Program.
-          </Text>
-          <Button size="large" icon={<SvgActionChevronR />} iconPlacement="right">
-            Sign up now
-          </Button>
-          <Text as="p" variant="t100" color="colorText" margin={{ top: 2 }}>
-            It takes 3 minutes and is 100% free.
-          </Text>
-        </GridItem>
-      </LayoutGrid>
-      <HeroImageWrapper>
-        <BackImage srcSet={`${yt576} 576w, ${yt864} 864w, ${yt1152} 1152w, ${yt2304} 2304w`} alt="Hero back" />
-        <FrontImage
-          srcSet={`${hero576} 576w, ${hero864} 864w, ${hero1152} 1152w, ${hero2304} 2304w`}
-          alt="Hero front"
-          parallaxPosition={frontImagePosition}
-        />
-      </HeroImageWrapper>
-    </StyledLimitedWidthContainer>
+    <ParallaxProvider>
```

Comment:
You have already added `ParallaxProvider` to `App.tsx`, do we need it here as well?